www/dfe_shiny_gov_style.css

```diff
@@ -56,6 +51,9 @@ a:active {
   text-decoration: underline
 }
 
```

Comment:
I've added some extra styling in the KS4RRS to put some padding in the cookie banner:
https://github.com/dfe-analytical-services/KS4-ready-reckoners/commit/54bf8e935b8dd601d7a5ea39ff1c5924b6499101